### ISSUE TITLE
Nothing works except highlighting

### ISSUE THREAD

**Issue by bajosiaa** [graphql/vscode-graphql#339]
## Actual Behavior
Nothing works except syntax highlighting.

## Expected Behavior
autocomplete suggestions
validation against schema
hover support
go to definition support (input, enum, type)

- GraphQL for VSCode Extension Version: 0.3.18
- VSCode Version: 1.62.2
- OS Name: Ubuntu
- OS Version: 20.04
*WSL2*

**Comment by acao** [comment]
Sample config file?

**Comment by bajosiaa** [comment]
```yml
// .graphqlrc.yml
schema: "src/graphql/schema.gql"
```

**Comment by acao** [comment]
ugh, there is still some kind of nasty bug. I think someone tried to fix schema paths with node.js file URIs and it possibly made things in the language server worse.

this is annoying, but can you try `schemaPath` instead of `schema`? and then restarting the language server?

**Comment by adnan-sheikh** [comment]
It was working fine for me until yesterday. Today, I can only see syntax highlighting and no autocompletions. Here's my config file:
```json
{
  "schema": "./graphql.schema.json",
  "documents": "./src/components/**/*.graphql"
}
```

**Comment by epegzz** [comment]
Same here. Stopped working for me. Probably today, not 100% sure.

```yaml
# .graphqlrc.yml
schema: ./graphql-schema.graphql

```

**Comment by adnan-sheikh** [comment]
> then restarting the language server?

how can I do this ? I really need this to work.

**Comment by acao** [comment]
It must be the new language server version broke SDL support 👎🏻 

You may just need to install an earlier version of the extension. I will see if I can unpublish

**Comment by adnan-sheikh** [comment]
v0.3.18 seems to work perfectly fine. I switched

**Comment by tonyfromundefined** [comment]
> v0.3.18 seems to work perfectly fine. I switched

I have same issue and issue was fixed when I downgraded to version `0.3.18`.

There is an extension `orsenkucher.vscode-graphql` that is forked with version `0.3.18`. So, I'm using it now.

**Comment by acao** [comment]
please try the latest release
